stable/mysql/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: mysql
-version: 0.15.0
+version: 0.15.1
```

Comment:
Since this introduce a new feature, the minor version should be bumped. After that, this one is good to go, I think

Reply:
addressed in 740130d3f35b65fc2d4ec55a6f72097c74a7d18f